backend/tracim_backend/lib/webdav/resources.py

```diff
@@ -892,14 +892,13 @@ def copyMoveSingle(self, destpath, isMove):
         except ContentNotFound:
             destination_parent = None
         try:
-            new_content = self.content_api.copy(
+            self.content_api.copy(
```

Comment:
you changed the behavior without to change the name. It's dangerous. The new copy is always recursive? You need to make this information explicit somewhere (probably in the `copy()` method docstring)

Reply:
done. It is always recursive now.